### DISCUSSION TITLE
Is default secret used?

### DISCUSSION THREAD

**Discussion by hyperknot** [ory/kratos#3029] [Q&A]
In the quickstarts, I see secret / cookie and cipher being used. I don't see `default` being used.

The ref document says this: "used for signing and encrypting **things**". Can you explain what are **things**?

**Comment by vinckr** [comment]
[The reference configuration](https://www.ory.sh/docs/kratos/reference/configuration) documents all keys, it is a reference for all possible configuration values.
That does not mean all of them are required or even useful. 
I am not sure what exactly the use of the `default` secret here is but I think you do not need it.
